### PR TITLE
Updating the torchvision version to make it work for python 3.10

### PR DESCRIPTION
**Pull request by kaushikb11**
# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

## What does this PR do?

Updating the torchvision version to make it work for python 3.10

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Review by awaelchli — APPROVED** [review]
Quick start might be running e2e on the lightning repo, just fyi

File: requirements.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 jsonargparse[signatures]
 wandb==0.12.16
 gradio==2.9.4
 pyyaml==5.4.0
 protobuf<4.21.0  # 4.21 breaks with wandb, tensorboard, or pytorch-lightning: https://github.com/protocolbuffers/protobuf/issues/10048
 lightning_hpo
 websockets
-lightning
\ No newline at end of file
+lightning
```
Comment (dmitsf) [inline requirements.txt:9]: Where is the difference here :) ?

**Review by dmitsf — APPROVED** [review]
🚀

File: requirements.txt (hunk 1 above)
Comment (awaelchli) [reply requirements.txt:9]: line endings and this repo does not use auto-formatter

**Pull request merged**